renderer/tdf/common/src/renderer/tdf/viewnode/view_node.cc

```diff
@@ -18,13 +18,17 @@
  * limitations under the License.
  */
 
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
```

Comment:
move to actual use place and add push & pop syntax

Reply:
fixed